src/twisted/test/proto_helpers.py

```diff
@@ -74,10 +81,20 @@ def connectionLost(self, reason):
             d.callback(None)
 
 
+
+deprecatedModuleAttribute(
+    Version("Twisted", 19, 3, 0),
```

Comment:
As per our [coding standard](https://twisted.readthedocs.io/en/twisted-19.2.0/core/development/policy/coding-standard.html?highlight=coding%20standard#versioning) and [the docs for Incremental that it links to](https://github.com/twisted/incremental#updating), the right way to spell these versions is `Version("Twisted", "NEXT", 0, 0)`, so it'll be picked up in the next release cycle.  According to [our current versioning scheme](https://calver.org), 19.3 would mean "released in March of 2019".

Reply:
OK, fixed.